docs/stoploss.md

```diff
@@ -87,17 +125,28 @@ This option can be used with or without `trailing_stop_positive`, but uses `trai
     trailing_only_offset_is_reached = True
 ```
 
-Simplified example:
+Configuration (offset is buyprice + 3%):
 
 ``` python
-    stoploss = 0.05
+    stoploss = -0.10
+    trailing_stop = True
+    trailing_stop_positive = 0.02
     trailing_stop_positive_offset = 0.03
     trailing_only_offset_is_reached = True
 ```
 
+For example, simplified math:
+
 * the bot buys an asset at a price of 100$
-* the stop loss is defined at 5%
-* the stop loss will remain at 95% until profit reaches +3%
+* the stop loss is defined at -10%
+* the stop loss would get triggered once the asset dropps below 90$
+* stoploss will remain at 90$ unless asset increases to or above our configured offset
+* assuming the asset now increases to 103$ (where we have the offset configured)
+* the stop loss will now be -2% of 103$ = 100,94$
```

Comment:
Maybe we should add a note here that says that the 100.94 are "locked in" ? 

No matter how price-action works - sells should not occur much below 100.94.